src/plugins/dropdown_buttons/plugin.js

```diff
@@ -13,7 +13,8 @@ Selectize.define('dropdown_buttons', function (options) {
 
 	function selectAllOptions() {
 		$.each(self.options, function (i, option) {
-			if (self.items.indexOf(option.value) === -1) self.addItem(option.value);
+			value = option.value.toString();
+			if (!self.items.includes(value)) self.addItem(option.value);
```

Comment:
We don't use the `value` variable after this, do we need it?

Also, what problem does it solve exactly?

Reply:
my bad, forgot to replace `option.value` with `value`
the `option.value` could be integer, while `self.items` is an array of strings - here is mismatch of types